lib/adapter_base.coffee

```diff
@@ -94,10 +95,10 @@ class Adapter
    * @return {Promise}
   ###
   renderFile: (file, opts = {}) ->
-    opts = _.clone(opts, true)
+    opts = clone(opts, true)
     (new File(file))
       .read(encoding: 'utf8')
-      .then _.partialRight(@render, _.extend({ filename: file }, opts)).bind(@)
+      .then partialRight(@render, Object.assign({ filename: file }, opts)).bind(@)
```

Comment:
Ah right, I checked the .travis.yml and assumed support was dropped already https://github.com/jescalan/accord/commit/84daec4c4fba1728a31c38fe0cdf818f9487f3a3